hil_testing/spirent_automation/state_check.sh

```diff
@@ -0,0 +1,28 @@
+ #!/bin/bash
```

Comment:
The leading space that shellcheck identifies here should be removed:
```suggestion
#!/bin/bash
```